json2cmake/__init__.py

```diff
@@ -19,7 +19,8 @@ def freeze(obj):
 
 
 def parsecommand(command, directory=os.curdir):
-    command = shlex.split(command)
+    if (isinstance(command, basestring)):
```

Comment:
This isn't compatible with Python 3. Fortunately, it turns out I've already previously broke Python 3 compatibility, so all's good.